### DISCUSSION TITLE
auto login feature

### DISCUSSION THREAD

**Discussion by Justin-KM** [mstaal/msal_streamlit_authentication#24] [Show and tell]
thanks for your code, I made some changes to fit my use case.

a new parameter was added to support auto login 
msal_authentication(
...
auto_login = False
...
)

when it set to True, no login/out button present, it will try to login automatically.

ref.
https://github.com/Justin-KM/msal-streamlit-authentication-options.git

**Comment by mstaal** [comment]
Hi Justin,

Do you still have your sample for me to see? I could not find it. :-)

Would like to adjust it accordingly.

Sincerely,
Michael


Den fre. 27. okt. 2023 kl. 03.29 skrev Justin-KM ***@***.***>:
